catalog/spatial/registry/registry-report-action-provider/src/main/java/org/codice/ddf/registry/report/action/provider/RegistryReportActionProvider.java

```diff
@@ -81,19 +81,14 @@ public RegistryReportActionProvider(String id) {
 
     private List<Action> processSubject(Metacard metacard) {
 
-        if (metacard.getAttribute(RegistryObjectMetacardType.REGISTRY_ID) == null
-                || StringUtils.isBlank(metacard.getAttribute(RegistryObjectMetacardType.REGISTRY_ID)
-                .getValue()
-                .toString())) {
+        if (!RegistryUtility.hasRegistryId(metacard)) {
```

Comment:
I renamed hasRegistryId to make it more accurate to the operations it actually performs. Used getRegistryId here.